packages/grafana-data/src/field/displayProcessor.ts

```diff
@@ -22,6 +22,10 @@ interface DisplayProcessorOptions {
    * Will pick 'dark' if not defined
    */
   theme?: GrafanaTheme;
+  /**
+   * Used by auto decimals logic
+   **/
+  tickSize?: number;
```

Comment:
is this used?

Reply:
Ah no, i did go down a much more complex route of modifying GrapNG and AxisBuiler to support the same logic flot has where it has the tick size (delta) to aid the decimals logic. But found a solution that did not need that so reverted all of those changes. Must have missed reverting this one